app/code/Magento/BundleGraphQl/Model/Resolver/BundleItems.php

```diff
@@ -76,10 +63,6 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
             $value[ProductInterface::SKU]
         );
 
-        $result = function () use ($value, $linkField) {
-            return $this->bundleOptionCollection->getOptionsByParentId((int)$value[$linkField]);
-        };
-
-        return $this->valueFactory->create($result);
+        return $this->bundleOptionCollection->getOptionsByParentId((int)$value[$linkField]);
```

Comment:
This was not supposed to be replaced because this was actual performance optimization via Deferred mechanism.

Reply:
Fixed